store/sql_channel_store.go

```diff
@@ -52,7 +52,7 @@ func ClearChannelCaches() {
 	channelByNameCache.Purge()
 }
 
-func NewSqlChannelStore(sqlStore *SqlStore) ChannelStore {
+func NewSqlChannelStore(sqlStore SqlStore) ChannelStore {
```

Comment:
What does `SqlStore` have?  is it a bunch of pointers?  If it's non-pointers it will copy everything no?